src/themes/default/components/core/blocks/SidebarMenu/SidebarMenu.vue

```diff
@@ -110,13 +110,15 @@
           >
             <sub-btn
               v-if="currentUser"
+              id="my-account-links"
               :name="$t('My account')"
               class="bg-cl-transparent brdr-none fs-medium-small"
             />
             <sub-category
               v-if="currentUser"
               :my-account-links="myAccountLinks"
-              :id="'foo'"
+              id="my-account-links"
+              :is-category="false"
```

Comment:
component `SubCategory` doesn't have prop `isCategory`. If you wanted to add it to `SubBtn` then I think we should keep it as `true`, because by default we want to prefetch category.

Reply:
Ok, now I get it. Thanks for update :)